core/bin/external_node/src/config/mod.rs

```diff
@@ -633,6 +633,9 @@ impl From<ExternalNodeConfig> for InternalApiConfig {
                 l2_weth_bridge: config.remote.l2_weth_bridge_addr,
             },
             bridgehub_proxy_addr: config.remote.bridgehub_proxy_addr,
+            // TODO set values
```

Comment:
A `TODO` should not go as is into PR: it should be either fixed or a tracking issue should be added.